homeworks/david.tereshchenko_DavidTereshchenko/GameProject/index.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="stylesheet" type="text/css" href="css/style.css">
+        <title>Game Project</title>
+    </head>
+    <body>
+        <div class="wrapper">
+            <header class="header"></header>
+            <main class="page">
+                <section class="main-section">
+                    <div class="main-contend">
+                        <span class="image">
+                            <img src="img/image.png" alt="workspace ">
+                        </span>
+                        <div class="about-text">
+                            <h1 class="about-me">About me</h1>
+                            <div class="text-information">
+                                <p>Hi! My name is David, and I'm a Junior Frontend Developer. I am already familiar with main Web Technologies like HTML, CSS, JavaScript and Git version control system.</p>
+                                <p>This page was developed during the course 'Frontend for beginners' from Masters Academy in 2023.</p>
+                                <p>This is a social project from MOCG company where I got an opportunity to work with Frontend mentors and to create my own small project for the portfolio. </p>
+                                <p>You can contact me via https://github.com/DavidTereshchenko and/or check out my GitHub.</p>
```

Comment:
Same here. If we have a link element, why don't use it?

Reply:
Okay, it was not stated in the task description, so, proly you do like to keep it as is